tests/foreman/cli/test_satellitesync.py

```diff
@@ -0,0 +1,628 @@
+        settings.
+        2. The exported ISO has been imported in org/satellite.
+
+        @status: Manual
+        """
+
+    @run_only_on('sat')
+    @stubbed()
+    @tier3
+    def test_negative_import_cv_iso(self):
+        """Export whole CV version as ISO in directory and Import nothing.
+
+        @feature: ISS - Import
+
+        @steps:
+
+        1. Export whole CV version as ISO to a directory specified in
+        settings.
+        2. Don't copy exported ISO to /var/www/html/pub/export directory.
+        3. Attempt to import this not copied ISO from some other
+        org/satellite.
+
+        @assert:
+
+        2. The exported iso is not imported due to non availability.
+        3. Error is thrown for non availability of CV version ISO to import.
+
+        @status: Manual
+        """
+
+    @run_only_on('sat')
+    @stubbed()
+    @tier3
+    def test_negative_export_cv_iso(self):
+        """Export whole CV version to iso is aborted due to insufficient
+        memory.
+
+        @feature: ISS - Export
+
+        @steps:
+
+        1. Attempt to Export whole CV version as iso to a directory which has
+        less memory available than contents size.
+
+        @assert:
+
+        1. The export CV version to iso has been aborted due to insufficient
+        memory.
+
+        @status: Manual
+        """
+
+    @run_only_on('sat')
+    @stubbed()
+    @tier3
+    def test_negative_export_cv_iso_max_size(self):
+        """Export whole CV version to iso is aborted due to inadequate maximum
+        iso size.
+
+        @feature: ISS - Export
+
+        @steps:
+
+        1. Attempt to Export whole CV version as iso with mb size less than
+        required.
+
+        @assert:
+
+        1. The export CV version to iso has been aborted due to maximum size is
+        not enough to contain the CV version contents.
+
+        @status: Manual
+        """
+
+    @run_only_on('sat')
+    @stubbed()
+    @tier3
+    def test_positive_export_cv_iso_max_size(self):
+        """CV version exported to iso in maximum iso size.
+
+        @feature: ISS - Export
+
+        @steps:
+
+        1. Attempt to Export whole CV version as iso with mb size more than
+        required.
+
+        @assert:
+
+        1. CV version has been exported to iso successfully.
+
+        @status: Manual
+        """
+
+    @run_only_on('sat')
+    @stubbed()
+    @tier3
+    def test_positive_export_import_cv_incremental(self):
+        """Export and Import CV version contents incrementally.
+
+        @feature: ISS - Export Import
+
+        @steps:
+
+        1. In upstream, Export CV version contents to a directory specified in
+        settings.
+        2. In downstream, Import these copied contents from some other
+        org/satellite.
+        3. In upstream, Add new packages to the CV.
+        4. Export the CV incrementally from the last date time.
+        5. In downstream, Import the CV incrementally.
+
```

Comment:
Upstream : The satellite/org which exports the CV/repo to be imported by downstream satellite/org.
Downstream: The satellite/org which imports the exported CV/repo from upstream satellite.